2025-05-06 23:08:43.975538: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:485] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
2025-05-06 23:08:43.994001: E external/local_xla/xla/stream_executor/cuda/cuda_dnn.cc:8454] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
2025-05-06 23:08:43.999798: E external/local_xla/xla/stream_executor/cuda/cuda_blas.cc:1452] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
2025-05-06 23:08:44.909325: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
I0506 23:08:49.022880 128567741466112 train.py:76] Training with config:
best_checkpoint_all_hosts: false
best_model_eval_metric: ''
best_model_eval_metric_higher_is_better: true
binary_args:
- !!python/tuple
  - --define cudnn_embed_so
  - 1
- !!python/tuple
  - --define=cuda_compress
  - 1
checkpoint_dir: /tmp/training/
checkpoint_interval_type: null
eval_initial_weights: false
eval_modes: !!python/tuple []
eval_specific_checkpoint_dir: ''
experiment_kwargs:
  config:
    dataset:
      name: kodak
      num_examples: 1
      num_frames: null
      root_dir: /home/shitty-gang/Desktop/Datasets/Kodak
      skip_examples: 0
      spatial_patch_size: null
      video_idx: null
    eval: {}
    log_gradient_norms: false
    log_per_datum_metrics: true
    loss:
      rd_weight: 0.001
      rd_weight_warmup_steps: 0
    model:
      entropy:
        activation_fn: gelu
        clip_like_cool_chic: true
        conditional_spec:
          interpolation: bilinear
          prev_kernel_shape: &id001 !!python/tuple
          - 3
          - 3
          use_conditioning: false
          use_prev_grid: false
        context_num_rows_cols: *id001
        layers: &id002 !!python/tuple
        - 18
        - 18
        scale_range: !!python/tuple
        - 0.001
        - 150
        shift_log_scale: 8.0
        use_linear_w_init: true
      latents:
        add_gains: true
        downsampling_exponents: !!python/tuple
        - 0
        - 1
        - 2
        - 3
        - 4
        - 5
        - 6
        downsampling_factor: 2.0
        gain_factor: null
        gain_values: null
        learnable_gains: false
        num_grids: 7
        q_step: 0.4
      quant:
        q_steps_bias:
        - 5.0e-05
        - 0.0001
        - 0.0005
        - 0.001
        - 0.003
        - 0.006
        - 0.01
        q_steps_weight:
        - 5.0e-05
        - 0.0001
        - 0.0005
        - 0.001
        - 0.003
        - 0.006
        - 0.01
      synthesis:
        activation_fn: gelu
        add_activation_before_residual: false
        add_layer_norm: false
        b_last_init_input_mean: false
        clip_range: !!python/tuple
        - 0.0
        - 1.0
        kernel_shape: 1
        layers: *id002
        num_residual_layers: 2
        residual_kernel_shape: 3
      upsampling:
        kwargs:
          interpolation_method: bilinear
        type: image_resize
    opt:
      cosine_decay_schedule: true
      cosine_decay_schedule_kwargs:
        alpha: 0.0
        decay_steps: 100000
        init_value: 0.01
      grad_norm_clip: 0.1
      max_num_ste_steps: 10000
      noise_log_every: 100
      num_noise_steps: 100000
      ste_break_at_lr: 1.0e-08
      ste_init_lr: 0.0001
      ste_log_every: 50
      ste_lr_decay_factor: 0.8
      ste_num_steps_not_improved: 20
      ste_reset_params_at_lr_decay: true
      ste_uses_cosine_decay: false
    quant:
      kumaraswamy_decay_steps: 100000
      kumaraswamy_end_value: 1.0
      kumaraswamy_init_value: 2.0
      noise_quant_type: soft_round
      soft_round_temp_end: 0.1
      soft_round_temp_start: 0.3
      ste_quant_type: ste_soft_round
      ste_soft_round_temp: 0.0001
      use_kumaraswamy_noise: true
interval_type: steps
legacy_random_seed_behavior: false
log_all_hosts: false
log_all_train_data: false
log_async: true
log_tensors_interval: 1
log_train_data_interval: 1.0
logging_interval_type: null
max_checkpoints_to_keep: 5
one_off_evaluate: false
periodic_action_growth_ratios: null
random_mode_eval: same_host_same_device
random_mode_train: unique_host_unique_device
random_seed: 0
save_checkpoint_interval: -1
save_initial_train_checkpoint: false
train_checkpoint_all_hosts: false
training_steps: 1

I0506 23:08:49.050438 128567741466112 xla_bridge.py:689] Unable to initialize backend 'rocm': NOT_FOUND: Could not find registered platform with name: "rocm". Available platform names are: CUDA
I0506 23:08:49.051279 128567741466112 xla_bridge.py:689] Unable to initialize backend 'tpu': INTERNAL: Failed to open libtpu.so: libtpu.so: cannot open shared object file: No such file or directory
I0506 23:08:49.447034 128567741466112 utils.py:309] [jaxline] experiment init starting...
E0506 23:08:49.448607 128567741466112 utils.py:314] [jaxline] experiment init failed with error.
Traceback (most recent call last):
  File "/home/shittyprogrammers/anaconda3/envs/c3/lib/python3.10/site-packages/jaxline/utils.py", line 311, in log_activity
    yield
  File "/home/shittyprogrammers/anaconda3/envs/c3/lib/python3.10/site-packages/jaxline/train.py", line 89, in train
    experiment = _initialize_experiment(
  File "/home/shittyprogrammers/anaconda3/envs/c3/lib/python3.10/site-packages/jaxline/train.py", line 48, in _initialize_experiment
    experiment = experiment_class(
  File "/home/shittyprogrammers/Desktop/C3-orignal/experiments/base.py", line 62, in __init__
    self._train_data_iterator = data_loading.load_dataset(
  File "/home/shittyprogrammers/Desktop/C3-orignal/utils/data_loading.py", line 414, in load_dataset
    ds = Kodak(root=root, transform=transform)
  File "/home/shittyprogrammers/Desktop/C3-orignal/utils/data_loading.py", line 116, in __init__
    os.makedirs(self.root)
  File "/home/shittyprogrammers/anaconda3/envs/c3/lib/python3.10/os.py", line 215, in makedirs
    makedirs(head, exist_ok=exist_ok)
  File "/home/shittyprogrammers/anaconda3/envs/c3/lib/python3.10/os.py", line 215, in makedirs
    makedirs(head, exist_ok=exist_ok)
  File "/home/shittyprogrammers/anaconda3/envs/c3/lib/python3.10/os.py", line 215, in makedirs
    makedirs(head, exist_ok=exist_ok)
  File "/home/shittyprogrammers/anaconda3/envs/c3/lib/python3.10/os.py", line 225, in makedirs
    mkdir(name, mode)
PermissionError: [Errno 13] Permission denied: '/home/shitty-gang'
Traceback (most recent call last):
  File "/home/shittyprogrammers/anaconda3/envs/c3/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/shittyprogrammers/anaconda3/envs/c3/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/shittyprogrammers/Desktop/C3-orignal/experiments/image.py", line 697, in <module>
    app.run(functools.partial(platform.main, Experiment))
  File "/home/shittyprogrammers/anaconda3/envs/c3/lib/python3.10/site-packages/absl/app.py", line 308, in run
    _run_main(main, args)
  File "/home/shittyprogrammers/anaconda3/envs/c3/lib/python3.10/site-packages/absl/app.py", line 254, in _run_main
    sys.exit(main(argv))
  File "/home/shittyprogrammers/anaconda3/envs/c3/lib/python3.10/site-packages/jaxline/utils.py", line 531, in inner_wrapper
    return f(*args, **kwargs)
  File "/home/shittyprogrammers/anaconda3/envs/c3/lib/python3.10/site-packages/jaxline/platform.py", line 132, in main
    train.train(experiment_class, config, checkpointer, writer)
  File "/home/shittyprogrammers/anaconda3/envs/c3/lib/python3.10/site-packages/jaxline/utils.py", line 662, in inner_wrapper
    return fn(*args, **kwargs)
  File "/home/shittyprogrammers/anaconda3/envs/c3/lib/python3.10/site-packages/jaxline/train.py", line 89, in train
    experiment = _initialize_experiment(
  File "/home/shittyprogrammers/anaconda3/envs/c3/lib/python3.10/site-packages/jaxline/train.py", line 48, in _initialize_experiment
    experiment = experiment_class(
  File "/home/shittyprogrammers/Desktop/C3-orignal/experiments/base.py", line 62, in __init__
    self._train_data_iterator = data_loading.load_dataset(
  File "/home/shittyprogrammers/Desktop/C3-orignal/utils/data_loading.py", line 414, in load_dataset
    ds = Kodak(root=root, transform=transform)
  File "/home/shittyprogrammers/Desktop/C3-orignal/utils/data_loading.py", line 116, in __init__
    os.makedirs(self.root)
  File "/home/shittyprogrammers/anaconda3/envs/c3/lib/python3.10/os.py", line 215, in makedirs
    makedirs(head, exist_ok=exist_ok)
  File "/home/shittyprogrammers/anaconda3/envs/c3/lib/python3.10/os.py", line 215, in makedirs
    makedirs(head, exist_ok=exist_ok)
  File "/home/shittyprogrammers/anaconda3/envs/c3/lib/python3.10/os.py", line 215, in makedirs
    makedirs(head, exist_ok=exist_ok)
  File "/home/shittyprogrammers/anaconda3/envs/c3/lib/python3.10/os.py", line 225, in makedirs
    mkdir(name, mode)
PermissionError: [Errno 13] Permission denied: '/home/shitty-gang'
